addons/project/__manifest__.py

```diff
@@ -89,14 +89,118 @@
+            'web/static/src/legacy/scss/kanban_dashboard.scss',
+            'web/static/src/legacy/scss/kanban_examples_dialog.scss',
+            'web/static/src/legacy/scss/kanban_column_progressbar.scss',
+            'web/static/src/legacy/scss/kanban_view.scss',
+
+            'base/static/src/scss/res_partner.scss',
+
+            # Form style should be computed before
+            'web/static/src/views/form/button_box/*.scss',
+
+            'web/static/src/legacy/action_adapters.js',
+            'web/static/src/legacy/debug_manager.js',
+            'web/static/src/legacy/legacy_service_provider.js',
+            'web/static/src/legacy/legacy_client_actions.js',
+            'web/static/src/legacy/legacy_dialog.js',
+            'web/static/src/legacy/legacy_load_views.js',
+            'web/static/src/legacy/legacy_promise_error_handler.js',
+            'web/static/src/legacy/legacy_rpc_error_handler.js',
+            'web/static/src/legacy/root_widget.js',
+            'web/static/src/legacy/systray_menu.js',
+            'web/static/src/legacy/systray_menu_item.js',
+            'web/static/src/legacy/backend_utils.js',
+            'web/static/src/legacy/utils.js',
+            'web/static/src/legacy/web_client.js',
+            'web/static/src/legacy/js/_deprecated/*',
+            'web/static/src/legacy/js/chrome/*',
+            'web/static/src/legacy/js/components/*',
+            'web/static/src/legacy/js/control_panel/*',
+            'web/static/src/legacy/js/core/domain.js',
+            'web/static/src/legacy/js/core/mvc.js',
+            'web/static/src/legacy/js/core/py_utils.js',
+            'web/static/src/legacy/js/core/context.js',
+            'web/static/src/legacy/js/core/misc.js',
+            'web/static/src/legacy/js/fields/*',
+            'web/static/src/legacy/js/services/data_manager.js',
+            'web/static/src/legacy/js/services/session.js',
+            'web/static/src/legacy/js/tools/tools.js',
+            'web/static/src/legacy/js/views/**/*',
+            'web/static/src/legacy/js/widgets/data_export.js',
+            'web/static/src/legacy/js/widgets/date_picker.js',
+            'web/static/src/legacy/js/widgets/domain_selector_dialog.js',
+            'web/static/src/legacy/js/widgets/domain_selector.js',
+            'web/static/src/legacy/js/widgets/iframe_widget.js',
+            'web/static/src/legacy/js/widgets/model_field_selector.js',
+            'web/static/src/legacy/js/widgets/model_field_selector_popover.js',
+            'web/static/src/legacy/js/widgets/ribbon.js',
+            'web/static/src/legacy/js/widgets/week_days.js',
+            'web/static/src/legacy/js/widgets/signature.js',
+            'web/static/src/legacy/js/widgets/attach_document.js',
+            'web/static/src/legacy/js/apps.js',
+            'web/static/src/legacy/js/env.js',
+            'web/static/src/legacy/js/model.js',
+            'web/static/src/legacy/js/owl_compatibility.js',
```

Comment:
I guess it is still needed.

Reply:
I don't have the time to continue to reduce the assets for project sharing, I will continue.